examples/cron/triggerbinding.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: cron-binding
+spec:
+  params:
+  - name: gitrevision
+    value: master
+  - name: gitrepositoryurl
+    value: https://github.com/tektoncd/triggers
```

Comment:
@ncskier @vtereso you can supply static values here as well right?

Reply:
Yup! This was actually something I came across when making this. Created https://github.com/tektoncd/triggers/issues/165.